docs/_docs/2_6_deployment.md

```diff
@@ -225,41 +238,128 @@ hudi:trips->connect --path /app/uber/trips
+
+```java
+show logfile metadata --logFilePathPattern <log_file_path>
+```
+
+ - `show logfile records` - Read records from log files.
+
+```java
+show logfile records --logFilePathPattern <log_file_path>
+```
+
+ - `show env` - Show spark launcher env by key
+ - `show envs all` - Show spark launcher envs.
+ - `compactions show all` - Shows all compactions that are in active timeline.
+ - `compaction show` - Shows compaction details for a specific compaction instant.
+ - `compactions showarchived` - Shows compaction details for specified time window
+ - `compaction showarchived` - Shows compaction details for a specific compaction instant
+ - `compaction run` - Run Compaction for given instant time
+ - `compaction schedule` - Schedule Compaction
+ - `compaction unschedule` - Unschedule Compaction
+ - `compaction unscheduleFileId` - UnSchedule Compaction for a fileId
+ - `compaction validate` - Validate Compaction
+ - `compaction repair` - Renames the files to make them consistent with the timeline as dictated by Hoodie metadata. Use when compaction unschedule fails partially.
+ - `repair addpartitionmeta` - Add partition metadata to a table, if not present
+ - `repair corrupted clean files` - repair corrupted clean files
+ - `repair deduplicate` - De-duplicate a partition path contains duplicates & produce repaired files to replace with
+ - `repair overwrite-hoodie-props` - Overwrite hoodie.properties with provided file. Risky operation. Proceed with caution!
+ - `savepoints show` - Show the savepoints
+ - `savepoint create` - Savepoint a commit
+ - `savepoint delete` - Delete the savepoint
+ - `savepoint rollback` - Savepoint a commit
+ - `savepoints refresh` - Refresh table metadata
+ - `cleans show` - Show the cleans
+ - `cleans run` - run clean
+ - `clean showpartitions` - Show partition level details of a clean
+ - `cleans refresh` - Refresh table metadata
+ - `sync validate` - validate hive sync. There are lot of options with this command like --mode, --sourceDb, --targetDb, etc.
+
+```java
+sync validate --hiveUser <user> --hivePass <password> --hiveServerUrl <hive_server_url>
+```
+
+ - `metadata create` - Create the Metadata Table if it does not exist
+ - `metadata delete` - Remove the Metadata Table
+ - `metadata init` - Update the metadata table from commits since the creation
+ - `metadata list-files` - Print a list of all files in a partition from the metadata
+ - `metadata list-partitions` - Print a list of all partitions from the metadata
+ - `metadata refresh` - Refresh table metadata
+ - `metadata set` - Set options for Metadata Table
+ - `metadata stats` - Print stats about the metadata
+ - `clustering run` - Run Clustering
+ - `clustering schedule` - Schedule Clustering
+ - `bootstrap index showmapping` - Show bootstrap index mapping
+ - `bootstrap index showpartitions` - Show bootstrap indexed partitions
+ - `bootstrap run` - Run a bootstrap action for current Hudi table
+ - `temps_show` - Show all views name
```

Comment:
do you know how to create a view in the first place. I spent few mins sometime back, but couldn't figure it out. would be good to give an example of creating a view and then querying/deleting on the view.

Reply:
Ok, actually I copied it from existing docs. Let me figure this out.

Reply:
@nsivabalan We have exposed a `createOrReplace` method in `TempViewProvider` interface which is currently getting used for creating temporary views. However we are only using it internally and have not exposed any user facing command to allow him to create a view himself. 
This view creation is primarily happening in CommitsCommand. You can check for `createView` cli option in commands exposed in this class to see where we are using it. 

May be I can simply add a comment mentioning the commit commands which user can check to see how the view is getting created. WDYT? cc @vinothchandar